docs/api/non_geo_datasets.csv

```diff
@@ -33,6 +33,7 @@ Dataset,Task,Source,License,# Samples,# Classes,Size (px),Resolution (m),Bands
 `ReforesTree`_,"OD, R",Aerial,"CC-BY-4.0",100,6,"4,000x4,000",0.02,RGB
 `RESISC45`_,C,Google Earth,-,"31,500",45,256x256,0.2--30,RGB
 `Rwanda Field Boundary`_,S,Planetscope,"NICFI AND CC-BY-4.0",70,2,256x256,4.7,RGB + NIR
+`SatlasPretrain`_,"C, R, S","NAIP, Landsat, Sentinel",ESA AND CC0-1.0 AND ODbL-1.0 AND CC-BY-4.0,302M,137,512,0.5--30,"SAR, MSI"
```

Comment:
* Task: possibly object detection and instance segmentation as well?
* License: ESA and "public domain" aren't valid SPDX license identifiers, but I'd rather list all than just say "various" or leave it blank? Depends on how the table gets formatted
* Resolution: is this correct, or was everything resampled to the nearest zoom level?

Reply:
@favyen2 can you double check these?

Reply:
The tasks are:
* Semantic segmentation
* Per-pixel regression
* Object detection (bounding box)
* Instance segmentation
* Classification

For resolution, images are up-sampled to the next higher zoom level, so in terms of the original image resolution, I think this is correct. NAIP is 60 cm/pixel, and Landsat B8 is 15 m/pixel, so to be more precise it could be `0.5--15` (if the range should include all the bands of the satellite images, then it should be `0.5--60` since Sentinel-2 has 60 m/pixel bands).

Reply:
Usually I just look at the most common resolution of the satellite, so Sentinel-2 is 10 m and Landsat is 30 m.